platform/windows/detect.py

```diff
@@ -196,7 +196,7 @@ def configure_msvc(env, manual_msvc_config):
 
     ## Compile/link flags
 
-    env.AppendUnique(CCFLAGS=['/MT', '/Gd', '/GR', '/nologo'])
+    env.AppendUnique(CCFLAGS=['/MT', '/Gd', '/GR', '/bigobj', '/nologo'])
```

Comment:
What is this and the `-Wa,-mbig-obj` for?

Reply:
The library is so big that the linker needs the bigobj extension in both msvc and mingw gcc.

Reply:
Well, you are removing a couple formats i guess, so is it still now the case?